android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -184,22 +189,29 @@ class OkHttpTest {
 
       var socketClass: String? = null
 
-      val clientCertificates = HandshakeCertificates.Builder()
-        .addPlatformTrustedCertificates()
-        .addInsecureHost(server.hostName)
-        .build()
+      val clientCertificates =
+        HandshakeCertificates.Builder()
```

Comment:
I don’t understand why it wrapped the constructor call?

Reply:
Looks like it started doing https://github.com/google/google-java-format/wiki/The-Rectangle-Rule